src/Network/Pusher.hs

```diff
@@ -195,23 +201,29 @@ handleWebhooks pusher webhookF =
             else Nothing)
     (const webhookF)
 
--- Handle webhooks asyncronously given their AppKey has a corresponding
--- AppSecret which the WebhookEv has been encrypted by.
+-- Handle webhooks given their AppKey has a corresponding AppSecret which the 
+-- WebhookEv has been encrypted by.
 handleWebhooks'
```

Comment:
It would help if this had a more descriptive name that made it clear how this function is different.